server/routes/index.ts

```diff
@@ -2,11 +2,13 @@ import { Router } from 'express'
 import type { Services } from '../services'
 import homeRoutes from './home'
 import spikeRoutes from './spikes'
+import alphaRoutes from './alpha/alphaRouter'
 
 export default function routes(services: Services): Router {
   const router = Router({ mergeParams: true })
   router.use(homeRoutes())
   router.use(spikeRoutes(services))
+  router.use('/alpha', alphaRoutes(services))
   // Add more routes here
```

Comment:
Can put them straight into the app without the '/alpha' prefix for me. e.g. `/activity-list`

Reply:
Just thought this would be easier to remove at some point - i dont see this stuff making it into beta in its current form

Reply:
will do tho